wavespectra/specarray.py

```diff
@@ -701,11 +658,9 @@ def partition(
         wsp_darr,
         wdir_darr,
         dep_darr,
+        swells=3,
```

Comment:
The default here of 3 looks sensible, and makes sense to reduce given the change from max_swells - just making sure it been a deliberate change.

Reply:
Yes it was deliberate change, we more commonly request up to 3 swells rather than 5.